src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -115,6 +115,19 @@ private abstract class AbstractTableDataSourceFindUsagesContext :
                 // After adding us as the source, the manager should immediately call into us to
                 // tell us what the data sink is.
                 Debug.Assert(_tableDataSink != null);
+
+                // https://devdiv.visualstudio.com/web/wi.aspx?pcguid=011b8bdf-6d56-4f87-be0d-0092136884d9&id=359162
```

Comment:
Should we instead just cite DevDiv VSO #359162 here?

Reply:
Ah, I see you are just moving this comment. Might still be good to remove this hyperlink.

Reply:
Links are good, but can be simpler:

```suggestion
                // https://devdiv.visualstudio.com/DevDiv/_workitems/edit/359162
```